src/js/modules/breakpointHelper.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @overview Breakpoint handler
+ *
+ * @module breakpointHelper
+ */
+
+export const getBreakpoints = () => {
+    const output = {};
+
+    const screenSizer = document.createElement('div');
+    screenSizer.classList.add('c-screen-sizer');
+
+    document.body.appendChild(screenSizer);
+
+    const breakpoints = window.getComputedStyle(document.querySelector('.c-screen-sizer'))
+        .getPropertyValue('content')
+        .replace(/"/g, '')
+        .replace(/'/g, '')
+        .split(',');
+
+    if (breakpoints.length === 1 && breakpoints[0] === '') {
+        // When there is no content
+        return output;
+    }
+
+    breakpoints.forEach(breakpoint => {
+        const [breakpointName, breakpointValue] = breakpoint.split(':');
+        output[breakpointName] = breakpointValue;
+    });
+
+    return output;
+};
+
+function pixelStringToInt (pixels) {
```

Comment:
Use arrow function to keep the style consistent with other functions:

```js
const pixelStringToInt = pixels => parseInt(pixels.slice(0, -2), 10);
```

Reply:
Looking at this code, you don't actually need the `slice()` part, `parseInt()` will ignore non-numerical characters, which means you can just use parseInt in the code below.